Dropdown/src/Dropdown.jsx

```diff
@@ -332,9 +411,10 @@ Dropdown.defaultProps = {
     size: "small",
     closeOnClick: true,
     enabled: true,
-    autoHide: false,
+    autoHide: true,
```

Comment:
I always wonder when boolean defaults change. Why here? Is there an impact on consumers?

Reply:
it fixes an issue with scrollbars. They were always visible even if not needed. That way, the library will take care of hide them.